src/utils/secretmanager.rs

```diff
@@ -0,0 +1,66 @@
+use async_trait::async_trait;
+use aws_config::{BehaviorVersion, Region};
+use aws_sdk_secretsmanager::Client as awsClient;
+
+use super::errors::Error;
+pub struct Client {
```

Comment:
Add space between the `use` statement and the struct.
Additionnally, choose a more suitable name for this struct. e.g.
```suggestion
pub struct AwsSecret {
```